setup.py

```diff
@@ -126,6 +126,7 @@ def download_mafft(kind="deb") -> Path:
         "multipledispatch",
         "python-codon-tables",
     ],
+    package_data={"seqlike": ["*.ttf"]},
```

Comment:
Note for myself: I think this is the fix. However, not 100% sure. Will need to pull down to check.

Reply:
Confirming the presence of `FreeMono.ttf` in the pip-installation location:

```
(base) root@codespaces_81c95a:/opt/conda/lib/python3.9/site-packages/seqlike# ls
AlignCommandline.py  SeqLike.py          __init__.py  alignment_commands.py  alphabets.py     draw_utils.py  utils
FreeMono.ttf         SeqLikeAccessor.py  __pycache__  alignment_utils.py     codon_tables.py  encoders.py
```

Reply:
Leaving another note for myself:

1. We can programmatically execute `python -m build`.
2. We can then unzip and untar the resulting `.tar.gz` file in `dist/`.
3. We can then assert that `FreeMono.ttf` exists in the distributed source dir.